src/unix/unix_internal.h

```diff
@@ -423,6 +441,44 @@ static inline tuple file_get_meta(file f)
     return f->f.type == FDESC_TYPE_REGULAR ? fsfile_get_meta(f->fsf) : f->meta;
 }
 
+static inline boolean file_is_readable(fdesc f)
```

Comment:
fdesc_is_{readable,writable}

Reply:
OK, changed the naming to fdesc_is_{readable,writable}.